package.json

```
@@ -5,62 +5,64 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "test": "jest",
+    "build:mobx-angular": "ng build mobx-angular",
+    "test": "ng test --code-coverage",
+    "test:ci": "ng test --watch=false --code-coverage",
```

Skips execution of unit tests on example projects:

```suggestion
    "test:ci": "ng test mobx-angular --watch=false --code-coverage",
```